src/dnx.common/include/trace_writer.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "xplat.h"
+
+namespace dnx
+{
+    class trace_writer
+    {
+    public:
+        trace_writer(bool verbose) : m_verbose(verbose)
+        {}
+
+        void write(const dnx::char_t* entry, bool verbose)
```

Comment:
Hard to tell if the default should be `true` or `false`. The idea is that errors are always written but debugging messages are _verbose_